README.md

```diff
@@ -190,6 +190,10 @@ The full API Reference is available here.
 * [setbucketlifecycle.go](https://github.com/minio/minio-go/blob/master/examples/s3/setbucketlifecycle.go)
 * [getbucketlifecycle.go](https://github.com/minio/minio-go/blob/master/examples/s3/getbucketlifecycle.go)
 
+### Full Examples : Bucket encryption Operations
```

Comment:
we need to add deletebucketencryption too

Reply:
ok